app/src/main/java/org/systers/mentorship/view/activities/FeedbackActivity.kt

```diff
@@ -156,16 +156,16 @@ class FeedbackActivity : BaseActivity(), View.OnClickListener {
 
         val builder = AlertDialog.Builder(this)
         //set title for alert dialog
-        builder.setTitle("Alert")
+        builder.setTitle(getString(R.string.alert))
         //set message for alert dialog
-        builder.setMessage("Are you sure you want to Discard?")
+        builder.setMessage(R.string.confirm_discard_msg)
         builder.setIcon(android.R.drawable.ic_dialog_alert)
         //performing positive action
-        builder.setPositiveButton("Yes"){dialogInterface, which ->
+        builder.setPositiveButton(R.string.yes){dialogInterface, which ->
             super.onBackPressed()
         }
         //performing cancel action
-        builder.setNeutralButton("Cancel"){dialogInterface , which ->
+        builder.setNeutralButton(R.string.cancel){dialogInterface , which ->
```

Comment:
same here , you forgot to add the string for this "Cancel" @somayaj . by the way great on your first contribution 👍

Reply:
thanks!